tests/e2e/local/kind/README.md

```diff
@@ -0,0 +1,69 @@
+# Benefits:
+1. Set up a local kind(Kubernetes in Docker) environment once and run E2E tests on local machine, so you can test and debug locally.
+1. No need to worry about kubernetes cluster setup.
+
+# Prereqs:
+1. Set up Istio Dev environment using https://github.com/istio/istio/wiki/Dev-Guide.
+
+1. Install
+  * [docker](https://docs.docker.com/)
+  * [kind](https://github.com/kubernetes-sigs/kind)
+
+You can run the following script to check/install of all pre-requisites, or use it as a reference to install them manually.
+(This requires installation of [Homebrew](https://brew.sh) on MacOS or debian based Linux distributions)
+
+```bash
+. ./install_prereqs.sh
+```
+
+# Steps
+## 1. Set up kind Environment
+```bash
+. ./setup_host.sh
+```
+
+## 2. Build Istio images
+Build images on your host machine:
+```bash
+. ./setup_test.sh
+```
+
+## 2. Run tests!
+You can issue test commands on your host machine.
+E.g.
+```bash
+cd $ISTIO/istio
+make e2e_simple E2E_ARGS="--use_local_cluster" HUB=localhost:5000 TAG=latest
+```
+Note the special arguments like **E2E_ARGS**, **HUB**, and **TAG**. They are required to run these tests with the local cluster and a local registry inside the VM. And you can run multiple E2E tests sequentially against the same VM.
+The script has a number of options available [here](../../README.md#options-for-e2e-tests)
+
+# Cleanup
+To destroy the cluster:
+```bash
+kind delete
```

Comment:
I've just broken these _today_ to save needing to break them later. All commands will be `kind [verb] [noun]` following other kubernetes tools, particularly eg `kops create cluster` etc. https://github.com/kubernetes-sigs/kind/pull/59

This will be `kind delete cluster` xref: https://github.com/kubernetes-sigs/kind/pull/59 https://github.com/kubernetes-sigs/kind/issues/56